glambie/util/mass_height_conversions.py

```diff
@@ -215,6 +234,25 @@ def _gigatonnes_to_meters_water_equivalent(variable: float, area: float, density
     return (1e6 * variable) / (area * density_of_water)  # 1e6 to convert area from km2 to m2
 
 
+def _meters_water_equivalent_to_gigatonnes(variable: float, area: float, density_of_water: float = 997) -> float:
```

Comment:
I don't think this needs to be a separate function, especially if it's private and called on only one line. Similarly with `_meters_water_equivalent_to_meters`. And I re-iterate my earlier comment about embedding units into variable names. Well done on typehinting `Iterable` though.

Reply:
I see your point, but that would mean changing it everywhere within this file (7 times), as they all have separate functions... I followed Sophies structure for mass unit conversions, not sure who reviewed that last time and proposed to do it that way

Reply:
I would recommend changing it everywhere in the file. Doing so will halve the length of the file and make things easier to read.

As a general point, if you find yourself writing almost all of a docstring more than once, you probably don't have a good code structure.